quantized_mesh_encoder/encode.py

```diff
@@ -17,7 +21,8 @@ def encode(
         indices,
         bounds=None,
         sphere_method=None,
-        ellipsoid=WGS84):
+        ellipsoid=WGS84,
+        extensions=[]):
```

Comment:
You should never use a mutable default argument for a function https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments